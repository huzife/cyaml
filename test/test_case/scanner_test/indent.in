- 
  key1: string1
  key2: string2
    string3
      string4
  key3:
                string5
   string6
- end
