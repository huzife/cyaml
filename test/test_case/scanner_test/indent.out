(BLOCK_SEQ_START)#
(BLOCK_ENTRY)#
(BLOCK_MAP_START)#
(KEY)#
(SCALAR, key1)#
(VALUE)#
(SCALAR, string1)#
(KEY)#
(SCALAR, key2)#
(VALUE)#
(SCALAR, string2 string3 string4)#
(KEY)#
(SCALAR, key3)#
(VALUE)#
(SCALAR, string5 string6)#
(BLOCK_MAP_END)#
(BLOCK_ENTRY)#
(SCALAR, end)#
(BLOCK_SEQ_END)#
